### DISCUSSION TITLE
Full application template

### DISCUSSION THREAD

**Discussion by kainszs** [MDAnalysis/mdanalysis#4547] [GSoC Discussions]
Hi,

I would like to submit the complete proposal to GSoC. I am confused about the PDF that should be uploaded as part of it. I have also read through the instructions for the mentors and it is implied that the contributor should also answer, among other things, what will take up his time in addition to GSoC. Are there any further questions or is there a template for the proposal?

Should I write it like a letter of motivation?

Best,
Kai

**Comment by orbeckst** [comment]
@MDAnalysis/gsoc-mentors can you help here?

**Reply by hmacdope** [reply]
@kainszs there is no template as to exactly what you should upload, but the following is a good guide: https://google.github.io/gsocguides/student/proposal-example-1

**Reply by kainszs** [reply]
Thank you!

**Comment by hmacdope** [comment]
@kainszs Did this answer your question? If so mark as resolved.
